src/app/modules/app/components/sidebar/sidebar.component.html

```diff
@@ -20,7 +21,7 @@
     </div>
   </div>
   <div class="sidebar__footer">
-    <section class="sidebar__section" (click)="onLogOut()">
```

Comment:
'onLogOut' Event should not be assigned to the section, instead assign it to the button.